README.md

```diff
@@ -1 +1,37 @@
-# kernel searcher's app
\ No newline at end of file
+# kernel searcher's app
+
+## Introduction
```

Comment:
it's okay to not have a readme, this app will be changing a bunch, can add readme later

Reply:
OK